docs/source/kedro_project_setup/session.md

```diff
@@ -36,4 +36,34 @@ You can provide the following optional arguments in `KedroSession.create()`:
 - `save_on_close`: A boolean value to indicate whether or not to save the session to disk when it's closed
 - `env`: Environment for the `KedroContext`
 - `extra_params`: Optional dictionary containing extra project parameters
-for the underlying `KedroContext`; if specified, this will update (and therefore take precedence over) parameters retrieved from the project configuration
+for the underlying **`KedroContext`**; if specified, this will update (and therefore take precedence over) parameters retrieved from the project configuration
+
+## `bootstrap_project` and `configure_project`
+```{image} ../meta/images/kedro-session-creation.png
```

Comment:
Is there a reason for using this style of image rather than the `! []()` markdown notation? Reason I ask is that it doesn't render on GitHub and I think it would help our readers to have visibility on the graphic here if possible (and you can add `alt` tags if you want them, so no need to use the `rst` image syntax -- AFAICT?).

Reply:
very good question! I don't have a good idea, it appears to me @astrojuanlu did something to all mermaid diagram with this new style. I also found it annoying it won't render in VSCode.

Reply:
Block-level images aren't really needed https://myst-parser.readthedocs.io/en/latest/syntax/images_and_figures.html#block-level-images I kept them for consistency, they can be turned into inline images